### PR TITLE
Update Copyright year

### LINKED ISSUE #630: Update copyright year in about page
## Describe the bug:-
The copyright year on the about page is still 2020.

## To Reproduce

1.  Go to 'https://learn.circuitverse.org/'
2.  Click on 'About'
3.  See the error

## Expected behavior
The year needs to be changed to the current year.

## Screenshots

Screenshot (848)

## Desktop (please complete the following information):

* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

### PR DESCRIPTION
**Pull request by Karolina-Sas**
Copyright year change

Fixes #630 

#### Changes done:
- [x] Copyright year changed to 2023

#### Screenshots
image

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

**Comment by Prerna-0202** [comment]
could you update 2022 to 2023 in `_config.yml` too??

**Comment by yashu6march** [comment]
Hello @Prerna-0202 I am new to open-source and would like to contribute to this issue. Can you please assign this issue to me?
Regards

**Comment by Prerna-0202** [comment]
> Hello @Prerna-0202 I am new to open-source and would like to contribute to this issue. Can you please assign this issue to me? Regards

Hey @yashu6march, This issue was fixed in this [pr](https://github.com/CircuitVerse/Interactive-Book/pull/643)
You can check out other open issues.

File: README.md [hunk 1]
```diff
@@ -193,4 +193,4 @@ In addition to these plugins, Interactive book also inherit's all the plugins us
 ---
 ### Create, Contribute, Learn, and succeed with CircuitVerse!!!
 
-Interactive-Book is &copy; 2020 by [CircuitVerse](https://circuitverse.org/)
+Interactive-Book is &copy; 2023 by [CircuitVerse](https://circuitverse.org/)
```
Comment (tachyons) [inline README.md:196]:
```suggestion
Interactive-Book is &copy; 2024 by [CircuitVerse](https://circuitverse.org/)
```

File: about.md [hunk 2]
```diff
@@ -8,11 +8,11 @@ description: ""
 
 
 # About Interactive Book
-Interactive-Book is &copy; 2020 by [CircuitVerse](https://circuitverse.org/).
+Interactive-Book is &copy; 2023 by [CircuitVerse](https://circuitverse.org/).
```
Comment (tachyons) [inline about.md:11]:
```suggestion
Interactive-Book is &copy; 2024 by [CircuitVerse](https://circuitverse.org/).
```

**Pull request merged**